src/backend/distributed/catalog/dependency.c

```diff
@@ -335,7 +432,7 @@ follow_get_dependencies_for_object(void *context, const Form_pg_depend pg_depend
  *    the object we will not try and distribute it.
  */
 static bool
-follow_order_object_address(void *context, const Form_pg_depend pg_depend)
+follow_order_object_address(void *context, Form_pg_depend pg_depend)
```

Comment:
I'm OK with that, just curiosity.

Reply:
`Form_pg_depend` is a typedef to a `FormData_pg_depend *`, the const applies therefore to the pointer and not to the content of the pointer.

`const Form_pg_depend` will be interpreted as `FormData_pg_depend const *` instead of `const FormData_pg_depend *` which was the intention. As a matter of fact my editor was warning me about it, otherwise I would not have known :)